core/spec.md

```diff
@@ -1117,6 +1122,42 @@ HTTP body.
 
 ---
 
+### Registry Well-Known Config
+
+The registry comes with a well known location `/.well-known/xregistry.json`,
+following [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615).
+
+It acts as a single entrypoint from where locations and capabilities of the
+xRegistry implementation can be retrieved. Since this URL is always the same,
+it can be used to find out whether xRegistry is supported and where to find the API.
+It also allows to support multiple versions of the xRegistry protocol at the same time
```

Comment:
...allows to support multiple versions...  ->  ..allows for support of multiple versions...

Reply:
```suggestion
It allows for support of multiple versions of the xRegistry protocol at the same time
```